packages/fx-core/src/question/inputs/CreateProjectInputs.ts

```diff
@@ -29,7 +29,8 @@ export interface CreateProjectInputs extends Inputs {
     | "link-unfurling"
     | "search-app"
     | "collect-form-message-extension"
-    | "copilot-plugin-capability";
+    | "copilot-plugin-capability"
+    | "search-message-extension";
```

Comment:
Is this change dedicated for e2e test?

Reply:
Yes.

Reply:
What's search-app in L30? Do we leverage that option or clean it up?

Reply:
We cannot leverage it because the option id for search template in VSCode and in VS are different. The root cause for this is these 2 templates' names are different.